### DISCUSSION TITLE
FYI: Chrome Bug - PiP API stopped working on localhost breaking Popup

### DISCUSSION THREAD

**Discussion by samydoesit** [nuxt/devtools#483] [General]
In case anyone else notices this:
Unfortunately, the PiP (Picture in Picture API) has broken in Chrome under `http://localhost`.

`Uncaught (in promise) DOMException: Failed to execute 'requestWindow' on 'DocumentPictureInPicture': Opening a PiP window requires https or file protocol`

There is also an official issue about this in the Google Chrome Bugtracker:
https://bugs.chromium.org/p/chromium/issues/detail?id=1486404&q=component%3ABlink%3EMedia%3EPictureInPicture%20localhost&can=2

<img width="845" alt="image" src="https://github.com/nuxt/devtools/assets/15785987/050eaac3-77c5-445f-8b42-63632f8d51a0">

**Comment by tarushnagpal** [comment]
As a workaround you can develop on chrome 116 which was the first version which released pip for now, you can find it here (for mac) https://google-chrome.en.uptodown.com/mac/post-download/106936027

**Comment by arashsheyda** [comment]
this is not a bug, you need to run your dev server with `https` flag:

 #510
